sovtoken/sovtoken/test/helpers/helper_request.py

```diff
@@ -145,43 +134,46 @@ def schema(
         schema_request_future = build_schema_request(sdk_wallet_did, schema_data)
         schema_request = self._looper.loop.run_until_complete(schema_request_future)
         request = self._sdk.sdk_json_to_request_object(json.loads(schema_request))
+        request = self._sign_sdk(request, sdk_wallet=sdk_wallet)
         return request
 
     def _find_wallet_did(self, sdk_wallet):
         sdk_wallet = sdk_wallet or self._steward_wallet
         _, sdk_wallet_did = sdk_wallet
         return sdk_wallet_did
 
-    def payment_signatures(self, inputs, outputs):
-        """ Generate a list of payment signatures from inptus and outputs. """
-        signatures = []
-        inputs = self._prepare_inputs(inputs)
-        outputs = self._prepare_outputs(outputs)
-
-        for utxo in inputs:
-            to_sign = [[utxo], outputs]
-            signer = self._wallet.get_address_instance(utxo[ADDRESS]).signer
-            signature = signer.sign(to_sign)
-            signatures.append(signature)
-
-        return signatures
-
     def _prepare_outputs(self, outputs):
         return [
-            {ADDRESS: output[ADDRESS], AMOUNT: output[AMOUNT]}
+            {"recipient": output[ADDRESS], AMOUNT: output[AMOUNT]}
             for output in outputs
         ]
 
     def _prepare_inputs(self, inputs):
-        return [
-            {ADDRESS: utxo[ADDRESS], SEQNO: utxo[SEQNO]}
+        inps = [
+            utxo["source"]
             for utxo in inputs
         ]
 
+        return inps
+
     def _create_request(self, payload, identifier=None):
         return Request(
             reqId=Request.gen_req_id(),
             operation=payload,
             protocolVersion=CURRENT_PROTOCOL_VERSION,
             identifier=identifier
         )
+
+    # Copied this over from another helper. Don't really know what it does.
+    def _sign_sdk(self, request, sdk_wallet=None):
+        request = self._sdk.sdk_sign_request_objects(
+            [request],
+            sdk_wallet=sdk_wallet
+        )[0]
+
+        request = self._sdk.sdk_json_to_request_object(json.loads(request))
+
+        if request.signatures is None:
+            request.signatures = {}
```

Comment:
Why do we need this?

Reply:
Seems like these two strings are not needed at all. I can remove them now or later.